### ISSUE TITLE
Rename the project

### ISSUE THREAD

**Issue by bengaywins** [kraxarn/spotify-qt#58]
I'm going to give this a try, but legit, I think the project should be renamed. To something _not_ infringing on Spotify's trademark. That way you avoid having to rename if the project gains a lot of traction and Spotify decides to take legal action.

**Comment by kraxarn** [comment]
The naming is mostly inspired by [spotify-tui](https://github.com/Rigellute/spotify-tui), as it uses a very similar naming. Interesting things to take from the [design guidelines](https://developer.spotify.com/documentation/general/design-and-branding) though: *"We have a few pointers to consider when naming your app: [...] It cannot begin with “Spot” or be similar to “Spotify” in sound or spelling."* Which this, and other similar clients, clearly break. Maybe Spotify doesn't really care with non-profit open source projects? It's still definitely something to keep in mind and consider however.

**Comment by bengaywins** [comment]
Yeah I don't consider lax enforcement as motivation to continue on the path. I think a lot of it is likely as a result of certain parties being unaware and some within Spotify being okay with it. Which puts Spotify themselves at risk of losing their trademark if they don't enforce what they know about. And I really don't want to see a good project needing to rebrand. It's a tough spot to be in when it happens.
